### ISSUE TITLE
feat: Clickable links in YouTube shorts comments 

### ISSUE THREAD

**Issue by remenraj** [ReVanced/revanced-patches#2980]
### Application

YouTube

### Issue

Making links in comments of yt shorts videos clickable. 

### Patch

YouTube recently made links in comments section of shorts videos not clickable. Requesting patch to make the links clickable again. 

### Motivation

Patch will help with productivity as the links can be clicked. 

### Acknowledgements

- [x] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.

**Comment by SodaWithoutSparkles** [comment]
FYI: YouTube decided to make them unclickable because of the recent flood of scam videos. The links claimed to offer "free robux" and required you to complete a survey.

**Comment by DavidVTurley** [comment]
Yes still. I think for people who use revanced are smart enough to not click the link. It shoul at least be an option to have. Not having it makes using the commen/description box useless.
